src/main/java/edu/uc/cech/agrawadv/eventorganizer/user/User.java

```diff
@@ -3,6 +3,7 @@
 import java.util.UUID;
 
 public class User {
+	
```

Comment:
Added spacing because the rest of your code has a one line above and below the code for readability.  ref :https://css-tricks.com/write-better-code-3-levels-code-consistency/